frontend/src/components/account-mgmt/ManageUserDialog.vue

```diff
@@ -10,8 +10,8 @@
             <v-col cols="12" md="3">
               <AppLabel for="bceid">BCeID:</AppLabel>
             </v-col>
-            <v-col v-if="isAddingUser" cols="12" md="9">
-              <v-text-field id="bceid" v-model="user.userName" placeholder="BCeID" variant="outlined" density="compact" :rules="rules.required" :disabled="isLoading"></v-text-field>
+            <v-col v-if="isAddingUser && !wasNewUserAdded" cols="12" md="9">
+              <v-text-field id="bceid" v-model="user.userName" @blur="checkBCeIDExists(user.userName)" placeholder="BCeID" variant="outlined" density="compact" :rules="rules.required" :error-messages="errorMessages" :disabled="isLoading"></v-text-field>
```

Comment:
Is there any benefit to doing this onblur as opposed to a Save validation? I guess it we did it on Save it might get confusing if the user hits both this validation condition and the warning condition.

Reply:
I thought it was worth getting the message to the user sooner, as it could change their course their input actions (i.e. save them from entering redundant info).

Reply:
Agreed.